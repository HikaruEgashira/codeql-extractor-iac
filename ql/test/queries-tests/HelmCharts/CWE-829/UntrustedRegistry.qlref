security/HelmCharts/CWE-829/UntrustedRegistry.ql