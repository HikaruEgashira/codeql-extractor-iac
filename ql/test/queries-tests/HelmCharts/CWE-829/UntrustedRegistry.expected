| Chart.yml:9:14:9:25 | busybox:1.28 | untrusted container registry |
| Chart.yml:12:14:12:40 | ghcr.io ... ox:1.28 | untrusted container registry |
| Chart.yml:15:14:15:65 | ACCOUNT ... ox:1.28 | untrusted container registry |
| Chart.yml:19:14:19:40 | gchr.io ... ox:1.28 | untrusted container registry |
